### DISCUSSION TITLE
Why created so many virtual network interface are every time when I boot genymotion

### DISCUSSION THREAD

**Discussion by tongtongstudio** [Genymobile/feedback#46] [Genymotion Desktop] [answered]
![image](https://user-images.githubusercontent.com/35002905/218300554-f9cf78bf-c6d7-4d84-92db-222334f940eb.png)

**Comment by jmaneyrol69** [comment, marked as answer]
@tongtongstudio this discussion board is not suitable for support queries. Please contact our support team via [our help center](https://support.genymotion.com/hc/en-us/categories/201540645-Genymotion-Desktop).
Thank you.
